com.stremio.Stremio.appdata.xml

```diff
@@ -0,0 +1,5 @@
+  <content_rating type="oars-1.1">
```

Comment:
This lacks at least the component tag along with the id, the rest can be added later

Reply:
I generated this using: https://hughsie.github.io/oars/generate.html

So i assumed that would have everything in there?

Reply:
Oh, the appdata needs more tags than just the OARS one. The specs are documented here https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps

Reply:
I've updated the appdata fie, i just need your email for the update contact, i can add mine though if you are too busy? :)

Reply:
Perfect, sure, you can use mine: bil[dot]elmoussaoui[at]gmail[dot]com. The only missing part is to install the appdata file in `$prefix/$datadir/metainfo`
Btw, can you rename the appdata file to `$app-id.metainfo.xml` ? New apptadata files should use the metainfo extension prefix instead.